vendor/github.com/cyphar/filepath-securejoin/.travis.yml

```diff
@@ -4,10 +4,12 @@
 
 language: go
 go:
-    - 1.7.x
-    - 1.8.x
+    - 1.13.x
+    - 1.16.x
     - tip
-
+arch:
+    - AMD64
+    - ppc64le
 os:
     - linux
     - osx
```

Comment:
No reason to update this, in fact it may confuse people.  This file can just "go away", we don't use travis anymore.

Reply:
Ugh, it's a vendor file, nevermind.